src/eigsolve/arnoldi.jl

```diff
@@ -475,10 +475,5 @@ function _restorearnoldiform!(U, H, f, keep)
         rmul!(view(H, 1:j, :), h')
         rmul!(U, h')
     end
-    copyto!(Hcompact, H) # copy back into compact Hessenberg form
-
-    # Update basis B by applying U
-    basistransform!(B, view(U, :, 1:keep))
-    B[keep + 1] = scale!!(r, 1 / βr)
```

Comment:
Oh right I forgot to remove these. I had so far only tested `_bischursolve` with `maxiter=1` before pushing, so I hadn't tested this part yet indeed. Thanks for fixing.